src/Discord.Net.Core/Discord.Net.Core.csproj

```
@@ -14,15 +14,10 @@
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="System.Interactive.Async" Version="6.0.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
-    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.4" />
+    <PackageReference Include="System.Interactive.Async" Version="6.0.3" />
```

Microsoft recommends replacing `System.Interactive.Async` with `System.Linq.AsyncEnumerable` (which is now part of .NET 10, and is provided to older versions via a NuGet package)
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/10.0/asyncenumerable
It's a prerelease version but it should be safe to replace.
```suggestion
    <PackageReference Include="System.Linq.AsyncEnumerable" Version="10.0.0-rc.1.25451.107" />
```

---

well it seems it isn't a simple drop in replacement. I'm gonna have to refactor some things.

---

Looks like [the method](https://github.com/dotnet/reactive/blob/1ce85f4066360193cea9da5b94def4c32552c3be/Ix.NET/Source/System.Linq.Async/System/Linq/Operators/ForEach.cs#L30) was not transferred to the new package, which makes sense.